corehq/apps/reports/templates/reports/tableau_visualization.html

```diff
@@ -10,6 +10,7 @@ <h2>{% trans "Tableau Visualizations" %}</h2>
 
 {% block pagination_templates %}
   <script type="text/html" id="tableau-visualization-template">
+    <td data-bind="text: title"></td>
```

Comment:
Optional: if you do want to control the column widths, you'd do it here, by adding a class like `col-md-1` to the cell that contains the delete button. See docs here: https://getbootstrap.com/docs/3.4/css/#grid-example-basic